Assets/MixedRealityToolkit/Services/MixedRealityPlayspace.cs

```diff
@@ -0,0 +1,90 @@
+﻿using UnityEngine;
+using System;
+using System.Collections.Generic;
+using Microsoft.MixedReality.Toolkit.Core.Utilities;
+
+public static class MixedRealityPlayspace
```

Comment:
Why put this in services if it doesn't have a service implementation?
Would this actually be just a utility class?

Reply:
I moved it over to Utilities. Left is the name. None of the classes there have MixedReality in their names. But the Playspace, as an instance in the scene is best identified with it.
What's your suggestion?